samples/README.md

```diff
@@ -69,23 +69,44 @@ Package oeenclave-clang was not found in the pkg-config search path.
 Perhaps you should add the directory containing `oeenclave-clang.pc`
 ```
 
-#### Build samples
+#### Build and Run samples
 
-To build a sample, change directory to your target sample directory and run `make build` to build the sample
-and run `make run` to run it.
+##### GNU Make
+
+To build a sample using GNU Make, change directory to your target sample directory and run `make build` to build the sample.
+Then execute "make run" to run the sample.
 
 For example:
 
 ```bash
 username@yourVMname:~/openenclave/share/openenclave/samples$ cd helloworld/
-username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ ls enc  host  Makefile
-
 username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ make build
- ...
-username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ make run host/helloworldhost ./enc/helloworldenc.signed
+username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ make run
```

Comment:
@CodeMonkeyLeet had informed me that we should remove the prompts from sample shell code to keep them simple, but I don't recall what, precisely, the format should be.

Reply:
addressed